batch/vep/Dockerfile

```diff
@@ -0,0 +1,61 @@
+# Copyright 2018 Google Inc.  All Rights Reserved.
```

Comment:
Any reason why these can't live under batch/build_annotator and run_annotator dirs? Even if you want to have a separate directory for some reason, it's confusing why we have a separate "vep" directory even though the existing dirs are also running VEP, so consider renaming this to make its purpose clear. Also, please add this to batch/README.md.

Reply:
I actually thought about it yesterday and tried to merge these two but (i) the current separation of `build_annotator` and `run_annotator` in the old approach (ii) the structure of database files which is different between the two approaches (iii) the fact that the other approach uses dsub and we are not (iv) and that it is geared toward variants in BigQuery etc. makes it more confusing to mix the two approaches.

Eventually we should merge the two approaches and come up with a single solution that covers all cases we want to cover then delete the rest. But that is a non-trivial effort that I do not want to spend time on it right now. My plan is to gradually bring things to `vep` directory and delete the other two directories. Other batch annotators can be added beside `vep` in future.

Updated the batch/README.md file (but it is a little confusing for reasons above).

Reply:
I see. I think having the readme is good for now as users know that this is WIP. Once we have a better idea of how the annotation workflow will look like, we should revisit the folders here and create a unified view.

Reply:
Yes, we should unify these, but it is not just our annotation workflow that matters. If we really want this _fork_ to be something meaningful, i.e., its changes can be merged into the upstream Verily repo, we need to somehow cover other annotation scenarios before unifying all folders.